### PR TITLE
[User] Prevent MailChimp call if user update fails

### PR DESCRIPTION
**Pull request by yonsaber**
We were calling MailChimp even if the result of `update_with_password` is false, same for admin side

### CODE REVIEW

File: app/controllers/admin/users_controller.rb [hunk 1]
```diff
@@ -18,8 +18,8 @@ def edit
 
   def update
     @user = User.find(params[:id])
-    @user.update!(user_params)
```
Comment (james) [inline app/controllers/admin/users_controller.rb:21]: This is calling update! with a bang, which means if it fails, it should error, and therefore never proceed. Therefore this if does nothing.

Comment (yonsaber) [reply app/controllers/admin/users_controller.rb:21]: Will remove

**Comment by yonsaber** [comment]
@james updated

**Comment by james** [comment]
V nice, thanks!

**Review by james — APPROVED** [review]

**Pull request merged**